docker-build/lom-slave-bullseye/Dockerfile

```diff
@@ -0,0 +1,475 @@
+FROM debian:bullseye
+
+MAINTAINER gulv@microsoft.com
+
+COPY ["no-check-valid-until", "/etc/apt/apt.conf.d/"]
+COPY ["apt-multiple-retries", "/etc/apt/apt.conf.d/"]
+COPY ["sources.list.amd64", "/etc/apt/sources.list"]
+COPY ["rsyslog.conf", "/etc/"]
+COPY ["lom_start.sh", "/usr/local/bin/"]
+
+## Make apt-get non-interactive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Auto-Generated for buildinfo
+ARG SONIC_VERSION_CACHE
+ARG SONIC_VERSION_CONTROL_COMPONENTS
+COPY ["buildinfo", "/usr/local/share/buildinfo"]
+# Remove unused
+COPY vcache/ /sonic/target/vcache/lom-slave-bullseye
+RUN dpkg -i /usr/local/share/buildinfo/sonic-build-hooks_1.0_all.deb
+ENV IMAGENAME=lom-slave-bullseye
+ENV DISTRO=bullseye
+RUN pre_run_buildinfo lom-slave-bullseye
+
+RUN apt-get update && apt-get install -y \
+        apt-utils \
+        default-jre-headless \
+        openssh-server \
+        curl \
+        wget \
+        unzip \
+        gzip \
+        git \
+        build-essential \
+        libtool \
+        lintian \
+        sudo \
+        dh-make \
+        dh-exec \
+        kmod \
+        libtinyxml2-dev \
+        python-all \
+        python-dev \
+        python-setuptools \
+        python3 \
```

Comment:
Could we reduce this docker image size by removing non-needed dependencies ? 

Since we are not using any c or python dependencies currently, could we bring down the size of the image with debian and go runtime installed, instead of 5 GB docker image ?

Reply:
The goals would still be met:
- We will adhere to azure build pipeline (just that non-needed dependencies are removed)
- We will have consistency across team for using same dev environment.

Reply:
No need. Said that, you can remove ones that are not needed. But no gain.

Reply:
Are we not gaining 4.5 GB if the docker can be spawned with .5 GB size ?

Reply:
This is slave docker that runs in your dev environment.
It is transient and you need it only when you run make.
So size does not matter.
Your VM has  2TB
Please make sure,  docker uses /data for its workspace.